http/1.5/modules/ROOT/pages/http-load-static-res-task.adoc

```diff
@@ -1,29 +1,72 @@
-= To Load a Static Resource
+= Load a Static Resource Example - Mule 4
 :keywords: anypoint, connectors, transports
 :page-aliases: connectors::http/http-load-static-res-task.adoc
 
-. Create a new project.
-. From the Mule Palette, drag an HTTP Listener operation, to a new flow.
-. Configure the Listener:
-* Click + and set the default host, port and base path. For example, host = 0.0.0.0, port = 8081, and no base path.
-* In General, set Path to the URL you plan to use to start the app from the browser. For example, set Path to */trigger*.
-* Accept the other default Listener settings, and save the configuration.
+The following example illustrates how to configure the Anypoint Connector for HTTP (HTTP Connector) *Listener* source and *Load static resource* operation to serve up static content for use with HTTP, using the request path to lookup the resource.
+To accomplish this example, you must create the Mule app, configure an HTTP global element, run, and test the app with curl commands.
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+.Load a static resource flow
+image::http-loadresource-flow.png[Load a static resource flow]
+
+== Create the Mule Application
+
+To create the Mule flow:
+
+. In the *Mule Palette* view, select the HTTP *Listener* source and drag it on to the canvas. +
```

Comment:
For 'on to' or 'onto', the main difference between the two is that onto is about movement from one place to another. The word 'on' is not.